src/linear_eq.jl

```diff
@@ -87,3 +87,47 @@ function gauss_seidel_interval_static!{T, N}(x::MVector{N, Interval{T}}, A::MMat
     end
     x
 end
+
+function preconditioner_static1{T, N}(A::SMatrix{N, N, Interval{T}}, b::SVector{N, Interval{T}})
+
+    Aᶜ = mid.(A)
+    B = inv(Aᶜ)
+
+    return B*A, B*b
+
+end
+
+
+function gauss_seidel_interval_static1{T, N}(A::SMatrix{N, N, Interval{T}}, b::SVector{N, Interval{T}}; precondition=true, maxiter=100)
+
+    n = size(A, 1)
+    x = @MVector fill(-1e16..1e16, n)
```

Comment:
`@SVector`?

Reply:
I realized only `x` has to be modified, so I tried a version where `A` and `b` are `SVector`s, and that showed improvement.

Reply:
Please try a version where `x` is an `SVector` and you use `setindex` to "modify" it (i.e. replace `x` with a new `SVector`).

Reply:
Done.
I think this version gives the best benchmarks.